pkg/client_data/lib/package_api.dart

```diff
@@ -88,6 +88,24 @@ class Message {
   Map<String, dynamic> toJson() => _$MessageToJson(this);
 }
 
+@JsonSerializable()
+class PackageData {
```

Comment:
Let's give this a documentation comment that this is used in `listVersions` API.